wled00/wled.cpp

```diff
@@ -105,6 +105,10 @@ void WiFiEvent(WiFiEvent_t event)
       break;
     case SYSTEM_EVENT_ETH_DISCONNECTED:
       DEBUG_PRINT(F("ETH Disconnected"));
+      // BUGBUG -- Is it appropriate to mess with WiFi when ethernet was disconnected?
+      //           Even if necessary in the past, ethernet is now only configured once,
+      //           during initial setup.  Until it's initialized, this event shouldn't
+      //           occur. If it is, why need to do anything here?
```

Comment:
It may be necessary to reconnect WiFi if wired connection is somehow lost.
Reconnecting WiFi (if necessary) is the only way to get access to ESP once wired connection is lost apart from physical access.

Reply:
Ah... ok, that makes sense.  I'll make one more commit to remove this comment.  Thanks!